modules/rest-api/pages/rest-cluster-addnodes.adoc

```diff
@@ -66,13 +66,9 @@ Failure to authenticate with the cluster gives `401 Unauthorized`.
 
 If an attempt is made to specify a node to be added with the scheme `http://` and/or the suffix `8091`, the request fails with `400 Bad Request`, and the error-message `http is prohibited due to security reasons, please use https`.
 
-If the node certificate on the new node does not correctly specify the node-name as a Subject Alternative Name, one of the following occurs:
+If the node certificate on the new node does not correctly specify the node-name as a Subject Alternative Name, failure may occur, with a message of the following form: `Attention: Prepare join failed. Failed to establish TLS connection to 10.144.231.102:18091. TLS handshake failure (TLS client: In state wait_cert_cr at ssl_handshake.erl:2084 generated CLIENT ALERT: Fatal - Handshake Failure {bad_cert,hostname_check_failed})`.
```

Comment:
Message is like this:

_Attention: Prepare join failed. Unable to validate certificate on host: 127.0.0.1. Please make sure the certificate on this host contains host name '127.0.0.1' in Subject Alternative Name. Refer to Couchbase docs for more info on how to create node certificates._

Reply:
Ok, have added.

Reply:
Have changed accordingly.